packages/components/psammead-grid/src/index.jsx

```diff
@@ -0,0 +1,181 @@
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MAX,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+const groups = {
+  group0: {
+    min: null,
+    max: GEL_GROUP_0_SCREEN_WIDTH_MAX,
+    gutterSize: GEL_GUTTER_BELOW_600PX,
+    marginSize: GEL_MARGIN_BELOW_400PX,
+  },
+  group1: {
+    min: GEL_GROUP_1_SCREEN_WIDTH_MIN,
+    max: GEL_GROUP_1_SCREEN_WIDTH_MAX,
+    gutterSize: GEL_GUTTER_BELOW_600PX,
+    marginSize: GEL_MARGIN_BELOW_400PX,
+  },
+  group2: {
+    min: GEL_GROUP_2_SCREEN_WIDTH_MIN,
+    max: GEL_GROUP_2_SCREEN_WIDTH_MAX,
+    gutterSize: GEL_GUTTER_BELOW_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+  group3: {
+    min: GEL_GROUP_3_SCREEN_WIDTH_MIN,
+    max: GEL_GROUP_3_SCREEN_WIDTH_MAX,
+    gutterSize: GEL_GUTTER_ABOVE_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+  group4: {
+    min: GEL_GROUP_4_SCREEN_WIDTH_MIN,
+    max: GEL_GROUP_4_SCREEN_WIDTH_MAX,
+    gutterSize: GEL_GUTTER_ABOVE_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+  group5: {
+    min: GEL_GROUP_5_SCREEN_WIDTH_MIN,
+    max: null,
+    gutterSize: GEL_GUTTER_ABOVE_600PX,
+    marginSize: GEL_MARGIN_ABOVE_400PX,
+  },
+};
+
+const group4WrapperMaxWidth = `45.5rem`;
+// (6.75rem * 6) + 5*16px gutters = 728 = 45.5 rem
+
+const group5WrapperMaxWidth = `46.4rem`;
+// (2.95rem * 12) + 11*16px gutters = 742.4 = 46.4 rem
+
+const mediaQuery = ({ min, max, styles }) => {
+  if (min && max) {
+    return `
+      @media (min-width: ${min}) and (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  if (min) {
+    return `
+      @media (min-width: ${min}) {
+        ${styles}
+      }
+    `;
+  }
+  if (max) {
+    return `
+      @media (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  return '';
+};
+
+const gelMaxWidths = css`
+  margin: 0 auto;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    max-width: ${group4WrapperMaxWidth};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    max-width: ${group5WrapperMaxWidth};
+  }
+`;
+
+const group4MaxWidth = css`
+  margin: 0 auto;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    max-width: ${group4WrapperMaxWidth};
+  }
+`;
+
+const gridMediaQueries = ({
+  columns,
+  startOffset,
+  enableGelGutters,
+  enableGelMargins,
+}) => {
+  const selectedGroups = Object.keys(columns);
+  return selectedGroups.map(group =>
+    mediaQuery({
+      min: groups[group].min,
+      max: groups[group].max,
+      styles: `
+        width: initial;
+        grid-template-columns: repeat(${columns[group]}, 1fr);
+        grid-column-end: span ${columns[group]};
+      ${enableGelGutters ? `grid-column-gap: ${groups[group].gutterSize};` : ``}
+      ${enableGelMargins ? `padding: 0 ${groups[group].marginSize};` : ``}
```

Comment:
&& would be cleaner

Reply:
I tried this. However, since this is within a template literal, I get the undefined value returned in the CSS. So I'm sticking with the ternary here.  
<img width="314" alt="Screenshot of CSS in browser with false false grid-column-start: undefined" src="https://user-images.githubusercontent.com/3028997/65713989-2e692300-e092-11e9-86dc-26467c813003.png">

Reply:
It does work for me using &&:
```
  ${enableGelGutters && `grid-column-gap: ${groups[group].gutterSize};`}
  ${enableGelMargins && `padding: 0 ${groups[group].marginSize};`}
```